packages/clerk-js/src/ui/hooks/usePopover.ts

```diff
@@ -20,7 +20,7 @@ type UsePopoverProps = {
 export type UsePopoverReturn = ReturnType<typeof usePopover>;
 
 export const usePopover = (props: UsePopoverProps = {}) => {
-  const { bubbles = true, shoudFlip = true, outsidePress } = props;
+  const { bubbles = false, shoudFlip = true, outsidePress } = props;
```

Comment:
❓Is this related ?

Reply:
No it's not I must have started off another branch. I'll make the necessary corrections.